src/styles/blocks/card.scss

```diff
@@ -0,0 +1,122 @@
+.card {
+  background-color: $white-color;
+  border: 1px solid $gray-elements-color;
+  border-radius: 5px;
+  padding: 32px 0 16px;
+  padding-inline: 16px;
+  width: $card-width;
+
+  transition:
+    transform 300ms,
+    color 300ms,
+    background 300ms;
+
+  &:hover {
+    transform: scale(120%);
+  }
+
+  &__img {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    margin: 0 auto 40px;
+    width: 160px;
+    height: 134px;
+    background-position: center;
+  }
+
+  &__description {
+    display: flex;
+    flex-direction: column;
+    gap: 4px;
+    padding-bottom: 16px;
+  }
+
+  &__title {
+    text-decoration: none;
+    color: $main-color;
+    font-size: 12px;
+    font-weight: $medium-weight;
+    line-height: 18px;
+    transition: color 300ms;
+  }
+
+  &:hover .card__title {
+    color: #34568b;
+  }
+
+  &__code {
+    font-size: 10px;
+    line-height: 14px;
+    font-style: normal;
+    font-weight: $regular-weight;
+    color: $secondary-color;
+  }
+
+  &__reviews {
+    display: flex;
+    justify-content: space-between;
+    align-items: flex-end;
+    margin-bottom: 24px;
+    width: 100%;
+  }
+
+  &__reviews-count {
+    line-height: 14px;
+    font-size: 10px;
+    font-style: normal;
+    font-weight: $regular-weight;
+    color: $main-color;
+    display: block;
+  }
+
+  &__price {
+    display: flex;
+    align-items: flex-end;
+    justify-content: space-between;
+    margin-bottom: 16px;
+    width: 100%;
+  }
+
+  &__price-text {
+    line-height: 18px;
+    font-size: 12px;
+    font-weight: $regular-weight;
+    color: $secondary-color;
+  }
+
+  &__dollar {
+    font-size: 16px;
+    font-weight: $bold-weight;
+    line-height: 18px;
+    color: $main-color;
+  }
+
+  &__button {
+    height: 40px;
+    width: 100%;
+    padding-block: 12px;
+    display: block;
+    border-radius: 5px;
+    background: $blue-color;
+    border: 1px solid $blue-color;
+    color: $white-color;
+    text-decoration: none;
+    flex-shrink: 0;
+    font-size: 14px;
+    font-weight: $bold-weight;
+    line-height: 16px;
+    text-transform: uppercase;
+    font-family: inherit;
+    transition:
+      background 300ms,
+      color 300ms;
+  }
+
+  &:hover .card__button {
+    background: $white-color;
+    border-radius: 5px;
```

Comment:
The border-radius is being redefined on hover, which is unnecessary since it's already set on the button. You can remove this line to simplify the hover effect.